datapipelines/src/main/java/io/customer/datapipelines/plugins/CustomerIODestination.kt

```diff
@@ -0,0 +1,127 @@
+package io.customer.datapipelines.plugins
+
+import com.segment.analytics.kotlin.core.AliasEvent
+import com.segment.analytics.kotlin.core.Analytics
+import com.segment.analytics.kotlin.core.BaseEvent
+import com.segment.analytics.kotlin.core.Constants
+import com.segment.analytics.kotlin.core.GroupEvent
+import com.segment.analytics.kotlin.core.IdentifyEvent
+import com.segment.analytics.kotlin.core.ScreenEvent
+import com.segment.analytics.kotlin.core.Settings
+import com.segment.analytics.kotlin.core.System
+import com.segment.analytics.kotlin.core.TrackEvent
+import com.segment.analytics.kotlin.core.platform.DestinationPlugin
+import com.segment.analytics.kotlin.core.platform.EventPipeline
+import com.segment.analytics.kotlin.core.platform.Plugin
+import com.segment.analytics.kotlin.core.platform.VersionedPlugin
+import com.segment.analytics.kotlin.core.platform.plugins.DestinationMetadataPlugin
+import com.segment.analytics.kotlin.core.platform.policies.CountBasedFlushPolicy
+import com.segment.analytics.kotlin.core.platform.policies.FlushPolicy
+import com.segment.analytics.kotlin.core.platform.policies.FrequencyFlushPolicy
```

Comment:
I don't remember if we have project wide recommendation for `*` imports, but this definitely is good use case to introduce common settings to switch to `*` imports (can be considered later though).

Reply:
I think ktlin doesnt recommend it might not let you do it by default.